x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_weblogs/ml/mean_bytes_by_file_extension.json

```diff
@@ -0,0 +1,38 @@
+
+{
+  "groups": [],
```

Comment:
As before, worth considering adding the job to a common group for all 3 modules `kibana_sample_data`.